### DISCUSSION TITLE
User rights function

### DISCUSSION THREAD

**Discussion by soneger** [navidrome/navidrome#3134] [Ideas]
Can you add a user rights function, such as the download button can be turned off, not to let the download, thanks！！

**Comment by metalheim** [comment]
At the moment, you can only disable downloads for the whole server with this configuration option:
https://www.navidrome.org/docs/usage/configuration-options/#:~:text=EnableDownloads

**Comment by 4bgt** [comment]
I want to support this feature request. Would be neat if one could somehow manage downloads per user, would also work with an in-between role between admin and user, maybe.
